### PR TITLE
tests/resource/aws_ec2_client_vpn_endpoint: Add sweeper

### PR DESCRIPTION
**Pull request by bflad**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request

Previous output from acceptance testing:

```
--- FAIL: TestAccAwsEc2ClientVpnEndpoint_basic (5.71s)
    testing.go:538: Step 0 error: Error applying: 1 error occurred:
        	* aws_ec2_client_vpn_endpoint.test: 1 error occurred:
        	* aws_ec2_client_vpn_endpoint.test: Error creating Client VPN endpoint: ClientVpnEndpointLimitExceeded: Limit exceeded
```

Previously output from AWS CLI:

```console
$ aws ec2 describe-client-vpn-endpoints | jq '.ClientVpnEndpoints | length'
5
```

Output from sweeper:

```console
$ go test ./aws -v -sweep=us-west-2 -sweep-run=aws_ec2_client_vpn_endpoint -timeout 10h
...
2019/04/02 12:46:28 [INFO] Deleting Client VPN Endpoint: cvpn-endpoint-073bf6cb2791e2d37
2019/04/02 12:46:28 [INFO] Deleting Client VPN Endpoint: cvpn-endpoint-056499587fe3f59c6
2019/04/02 12:46:29 [INFO] Deleting Client VPN Endpoint: cvpn-endpoint-0e92f7d99d531d5ae
2019/04/02 12:46:31 [INFO] Deleting Client VPN Endpoint: cvpn-endpoint-0b4741c9520d69d36
2019/04/02 12:46:31 [INFO] Deleting Client VPN Endpoint: cvpn-endpoint-0c5fadd250f9ca007
2019/04/02 12:46:32 Sweeper Tests ran:
	- aws_ec2_client_vpn_endpoint
	- aws_directory_service_directory
ok  	github.com/terraform-providers/terraform-provider-aws/aws	9.578s
```

Output from AWS CLI:

```console
$ aws ec2 describe-client-vpn-endpoints | jq '.ClientVpnEndpoints | length'
0
```

### CODE REVIEW

File: aws/resource_aws_ec2_client_vpn_endpoint_test.go [hunk 1]
```diff
@@ -94,7 +155,7 @@ func TestAccAwsEc2ClientVpnEndpoint_withLogGroup(t *testing.T) {
 	})
 }
 
-func TestAccAwsEc2ClientVpnEndpoint_withDNSServers(t *testing.T) {
+func TestAccAwsEc2ClientVpnEndpoint_withDNervers(t *testing.T) {
```
Comment (nywilken) [inline aws/resource_aws_ec2_client_vpn_endpoint_test.go]: Is this rename intentional?

Comment (bflad) [reply aws/resource_aws_ec2_client_vpn_endpoint_test.go]: 😅 Great catch
